lib/presentation/widget/destination_picker/destination_picker_viewmodel.dart

```diff
@@ -104,8 +118,8 @@ class DestinationPickerViewModel extends BaseViewModel {
     store.dispatch(DestinationPickerBackToSharedSpaceAction());
   }
 
-  void backToUploadDestination() {
-    store.dispatch(DestinationPickerGoToUploadDestinationAction());
+  void backToChooseSpaceDestination() {
+    store.dispatch(GoToChooseSpaceAction(null));
```

Comment:
I think input param should be `Operation.none` instead of `null`

Reply:
`null` will make `reducer` use the last Operation when we back.

Reply:
I think you can put optional parameter in GoToChooseSpaceAction so you don't specify anything here